docs/tutorial/tut_multi_active_satellites.md

```diff
@@ -0,0 +1,176 @@
+phone number, then both the phone number, and the extension attribute will be considered a child dependent key. 
+
+##### Hashdiff (src_hashdiff)
+This is a concatenation of the payload (below) and the primary key. This allows us to 
+detect changes in a record (much like a checksum). For example, if a customer changes their name, the hashdiff 
+will change as a result of the payload changing. 
+
+##### Payload (src_payload)
+The payload consists of concrete data for an entity (e.g. A customer). This could be
+a name, a phone number, a date of birth, nationality, age, gender or more. The payload will contain some or all of the
+concrete data for an entity, depending on the purpose of the satellite. 
+
+##### Effective From (src_eff)
+An effectivity date. Usually called `EFFECTIVE_FROM`, this column is the business effective date of a multi active
+satellite record. It records that a record is valid from a specific point in time.
+If a customer changes their name, then the record with their 'old' name should no longer be valid, and it will no 
+longer have the most recent `EFFECTIVE_FROM` value. 
+
+##### Load date (src_ldts)
+A load date or load date timestamp. This identifies when the record was first loaded into the database.
+
+##### Record Source (src_source)
+The source for the record. This can be a code which is assigned to a source name in an external lookup table, 
+or a string directly naming the source system.
+
+!!! note
+    `LOAD_DATE` is the time the record is loaded into the database. `EFFECTIVE_FROM` is different, 
+    holding the business effectivity date of the record (i.e. When it actually happened in the real world) and will usually 
+    hold a different value, especially if there is a batch processing delay between when a business event happens and the 
+    record arriving in the database for load. Having both dates allows us to ask the questions 'what did we know when' 
+    and 'what happened when' using the `LOAD_DATE` and `EFFECTIVE_FROM` date accordingly. 
+
+### Setting up MAS models
+
+Create a new dbt model as before. We'll call this one `ma_sat_customer_details`.
+
+=== "ma_sat_customer_details.sql"
+
+    ```jinja
+    {{ dbtvault.ma_sat(var('src_pk'), var('src_cdk'), var('src_hashdiff'), var('src_payload'),
+                    var('src_eff'), var('src_ldts'), var('src_source'),
+                    var('source_model'))                                   }}
+    ```
+
+To create a MAS model, we simply copy and paste the above template into a model named after the MAS we
+are creating. dbtvault will generate a MAS using parameters provided in the next steps.
+
+MAS should use the incremental materialization, as we load and add new records to the existing data set. 
+
+We recommend setting the `incremental` materialization on all of your MAS using the `dbt_project.yml` file:
+
+=== "dbt_project.yml"
```

Comment:
We need to update this section in ALL of the tutorial steps not to include the `dbt_project.yml` example anymore. Might be best to link to the Metadata reference, with a short explanation in the tutorial step.